.github/workflows/comment-changes.yml

```diff
@@ -44,6 +46,7 @@ jobs:
           assignees: |
             AmmarAbouZor
           branch: documentation/aeruginous-comment-changes
+          branch-suffix: timestamp
```

Comment:
This is required in order to prevent fragments from being dropped if a new fragment was created but will create one PR per fragment.